crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -51,7 +51,7 @@ module sui_system::sui_system {
     use sui::table::Table;
     use sui_system::validator::Validator;
     use sui_system::validator_cap::UnverifiedValidatorOperationCap;
-    use sui_system::sui_system_state_inner::{Self, SystemParameters, SuiSystemStateInnerV2};
+    use sui_system::sui_system_state_inner::{Self, SystemParameters, SuiSystemStateInnerV2, protocol_version};
```

Comment:
I don't see this change used anywhere? Did we not emit a warning?

Reply:
I did not see one